<launch>
    <node name="template_sensor" pkg="ssf_updates" type="template_sensor" clear_params="true" output="screen">
            <remap from="ssf_core/imu_state_input" to="/auk/fcu/imu" />
	    <remap from="ssf_core/template_measurement" to="vicon/auk/auk" />

	    <rosparam file="$(find ssf_updates)/pose_sensor_fix.yaml"/>
    </node>
</launch>
